module05/test_calculator.py

```diff
@@ -0,0 +1,28 @@
+"""Test calculator"""
+import unittest
+import calculator
+
+class CalculatorTestCase(unittest.TestCase):
+    """Testcase for calculator functions"""
+    def test_add(self):
+        """Test add()"""
+        result = calculator.add(10, 20)
```

Comment:
Really nice -- short and sweet. You have caught on the tricky moment with calling the "calculator.<method>" which is a requirement of the test.py